spec/packages_spec.rb

```diff
@@ -36,7 +42,12 @@
               expect { chef_run }.to_not raise_error
             end
             it do
-              php_pkg = version.to_f < 7 ? prefix : "mod_#{prefix}"
+              php_pkg =
```

Comment:
Would this be more readable as a case statement as you have above?
```ruby
php_pkg =
  case pltfrm
    when CENTOS_8
      'php'
    when CENTOS_7
      "mod_#{prefix}"
    else
      prefix
end
```

Reply:
Kind of, I'll push up a change that's works properly but this exactly doesn't work.